test/integration/docker_build_test.exs

```diff
@@ -27,6 +27,7 @@ defmodule ExDockerBuild.Integration.DockerBuildTest do
       log =
         capture_log(fn ->
           assert {:ok, image_id} = DockerBuild.build(instructions, "")
+          ExDockerBuild.delete_image(image_id)
```

Comment:
please assert on the result of the delete image this can be failing and no one can notice it

Reply:
done